src/style.css

```
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 800;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  margin: 20px 0;
+}
+
+html {
+  font-family: Avenir-Book, Arial, sans-serif;
+}
+
+.search {
+  font-family: inherit;
+  height: 70px;
+  width: 100%;
+  padding: 25px 30px 23px 62px;
+  background: url("./images/Search.svg") no-repeat  26px, center, #fff;
+  background-size: 19px;
+  font-size: 16px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+}
+
+.small {
+  height: 42px;
+  padding: 12px 30px 10px 33px;
+  background: url("./images/Search.svg") no-repeat  13px, center, #fff;
+  background-size: 11px;
+  font-size: 14px;
+}
+
+.search:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search:focus {
+  outline: none;
+  background: url("./images/Search.svg") no-repeat 26px center, linear-gradient(180deg,#fff,#f6f6f7);
+  font-family: Avenir-Heavy, Arial, sans-serif;
```

```suggestion
  font-family: inherit;
```
and move it to default state

---

I can't remove the tag label in index.html because the linter shows a mistake.